bridges/opentracing/lib/opentelemetry/bridge/opentracing/scope_manager.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # A ScopeManager provides an API for interfacing with
+      # OpenTelemetry Tracers and Spans as OpenTracing objects
+      class ScopeManager
+        def initialize(tracer)
+          @tracer = tracer
+        end
+
+        # Activate the given span
+        #
+        # @param [Span] span An OpenTelemetry Span
+        # @return [Span]
+        def activate(span, finish_on_close: true)
+          Span.new(@tracer.with_span(span))
+        end
+
+        # Get the active span as a OpenTracingBridge::Span
+        #
+        # @return [Span]
+        def active
+          Span.new(@tracer.current_span)
```

Comment:
This should return the active `Scope` from an instance variable.

Reply:
Reading some more, it looks like the current Scope should be tracked in a thread-local variable rather than an instance variable. This is similar to what we do in `OpenTelemetry::Context`.